src/keyboard/KeyboardViewport.tsx

```diff
@@ -0,0 +1,153 @@
+import { FC, PropsWithChildren, useEffect, useRef } from "react";
+
+type KeyboardViewportType = PropsWithChildren<{
+  className?: string;
+}>;
+
+const KEYMAP_SCALE = "keymap:scale";
```

Comment:
Why are we renaming the key? This will make existing stored values invalid

Reply:
i should've marked it as a `TODO`, i didn't want to break anything that might be using the namespace. this can be renamed to what will be preferred.